numba/tests/test_mixed_tuple_unroller.py

```diff
@@ -1246,6 +1326,7 @@ def foo():
                     acc -= 1
                 else:
                     break
+            _check_is_tuple(a)
```

Comment:
NOTE: This check is added to all tests to ensure transformation of const-list to `tuple` occurs. Prior to this the tests would pass regardless of whether `literal_unroll` was used as the list type used in the test would generally resolve to e.g. `list(complex128)` and work fine in operations such as reduction.